types/consensus_data.go

```diff
@@ -49,13 +62,204 @@ type ConsensusData struct {
 	AggregateAndProof      *phase0.AggregateAndProof
 	SyncCommitteeBlockRoot phase0.Root
 	// SyncCommitteeContribution map holds as key the selection proof for the contribution
+	// TODO<olegshmuelov>: consider to use array instead of the map for ssz
 	SyncCommitteeContribution ContributionsMap
```

Comment:
we should consider to use array instead of the map for ssz.
in this scenario we have to use temporary ssz struct to convert the map to sorted array in order to have the same struct after marshal/unmarshal ssz